go-apps/meep-app-enablement/server/app-support/app-support.go

```diff
@@ -231,6 +267,22 @@ func applicationsConfirmTerminationPOST(w http.ResponseWriter, r *http.Request)
 		return
 	}
 
+	//look if subscription exist to process the Termination POST
+	found := false
+	//loop through appTerm map
+	for _, appTermSubscription := range appTerminationNotificationSubscriptionMap {
```

Comment:
Why do you need this map? Why not just get the subscriptions for the specific app instance ID from redis?

Reply:
maps are used by services to prevent extra calls to the DB... expected performance gains...